app/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,6 @@
 
     <uses-permission android:name="android.permission.WAKE_LOCK" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
-    <uses-feature android:name="android.hardware.type.watch" />
```

Comment:
removed the watch feature to make the app work

Reply:
Hmmm.  I assume this is related to the compileOnly removal?  Perhaps that is something specific to the group and/or has not been pushed to GitHub.